core/lowering/lowering.cpp

```diff
@@ -60,10 +60,9 @@ void LowerGraph(std::shared_ptr<torch::jit::Graph>& g, LowerInfo lower_info) {
   LOG_GRAPH(*g);
 }
 
-torch::jit::Module LowerModule(
-    const torch::jit::Module& mod,
-    std::string method_name,
-    std::unordered_set<std::string> forced_fallback_modules) {
+torch::jit::Module LowerModule(const torch::jit::Module& mod, std::string method_name, const LowerInfo& lower_info) {
+  std::unordered_set<std::string> forced_fallback_modules(
+      lower_info.forced_fallback_modules.begin(), lower_info.forced_fallback_modules.end());
   passes::NotateModuleForFallback(mod, "", method_name, forced_fallback_modules);
```

Comment:
When the forced_fallback_modules is empty, we don't need to have this pass so we can disable it. This can prevent iterating through all the nodes in graph. Disabling this also fixed BERT issue but your other fix is necessary.